ccd-definition/CaseEvent/CareSupervision/MultiState.json

```diff
@@ -312,7 +313,8 @@
     "ShowSummary": "Y",
     "ShowEventNotes": "N",
     "EndButtonLabel": "Save and continue",
-    "CallBackURLSubmittedEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/callback/update-task-list/submitted"
+    "CallBackURLSubmittedEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/callback/update-task-list/submitted",
+    "EventEnablingCondition": "otherOrderType=\"NO\""
```

Comment:
hmm, does not looks right, unless i missed something. Flag otherOrderType is set only on one particular event. Before event is saved, then  otherOrderType is unspecified and events wont be visible (I think they should). Should it not be 'otherOrderType!=\"YES\"' ?

Reply:
Changed it to  'otherOrderType!="YES"'